include/deal.II/fe/fe_nedelec.h

```diff
@@ -42,14 +42,52 @@ DEAL_II_NAMESPACE_OPEN
  * consistent orientation of faces. See the todo entries below for more
  * detailed caveats.
  *
- * Implementation of N&eacute;d&eacute;lec elements, conforming with the space
- * H<sup>curl</sup>. These elements generate vector fields with tangential
- * components continuous between mesh cells.
+ * Implementation of N&eacute;d&eacute;lec elements. The N&eacute;d&eacute;lec
+ * space is designed to solve problems in which the solution only lives in the
+ * space
+ * $H^\text{curl}=\{ {\mathbf u} \in L_2: \text{curl}\, {\mathbf u} \in L_2\}$,
+ * rather than in the more commonly used space
+ * $H^1=\{ u \in L_2: \nabla u \in L_2\}$. In other words, the solution must
+ * be a vector field whose curl is square integrable, but for which the
+ * gradient may not be square integrable. The typical application for this
+ * space (and these elements) is to the Maxwell equations and corresponding
+ * simplifications, such as the reduced version of the Maxwell equation
+ * that only involves the electric field $\mathbf E$ which has to satisfy
+ * the equation $\text{curl}\, \text{curl}\, {\mathbf E} = 0$ in the
+ * time independent case when no currents are present, or the equation
+ * $\text{curl}\,\text{curl}\,{\mathbf A} = 4\pi{\mathbf j}$ that the
+ * magnetic vector potential $\mathbf A$ has to satisfy in the
+ * time independent case.
  *
- * We follow the convention that the degree of N&eacute;d&eacute;lec elements
- * denotes the polynomial degree of the largest complete polynomial subspace
- * contained in the N&eacute;d&eacute;lec space. This leads to the
- * consistently numbered sequence of spaces
+ * The defining
+ * characteristic of functions in $H^\text{curl}$ is that they are in
+ * general discontinuous -- but that if you draw a line in 2d (or a
+ * surface in 3d), then the <i>tangential</i> component(s) of the vector
+ * field must be continuous across the line (or surface) even though
+ * the normal component may not be. As a consequence, the
+ * N&eacute;d&eacute;lec element is constructed in such a way that (i) it is
+ * @ref vector_valued "vector-valued", (ii) the shape functions are
+ * discontinuous, but (iii) the tangential component(s) of the vector field
+ * represented by each shape function are continuous across the faces
+ * of cells.
+ *
+ * Other properties of the N&eacute;d&eacute;lec element are that (i) it is
+ * @ref GlossPrimitive "not a primitive element"; (ii) the shape functions
+ * are defined so that certain integrals over the faces are either zero
+ * or one, rather than the common case of certain point values being
+ * either zero or one. (There is, however, the FE_RaviartThomasNodal
```

Comment:
```suggestion
 * either zero or one.
```
I think you do not want to refer to RT here - do we happen to have a nodal H^curl element to mention here?

Reply:
No, it's really just a copy-paste error.